src/utils/storage/index.ts

```diff
@@ -1,54 +1,27 @@
-import { ImmortalStorage, LocalStorageStore } from 'immortal-db'
-
 import { getNetworkName } from 'src/config'
-import { Errors, logError } from 'src/logic/exceptions/CodedException'
+import Storage from './Storage'
 
-const stores = [LocalStorageStore]
-export const storage = new ImmortalStorage(stores)
+export const storage = new Storage(window.localStorage, '')
 
-// We need this to update on run time depending on selected network name
-export const getStoragePrefix = (networkName = getNetworkName()): string => `v2_${networkName}`
+// We need this to update the key in runtime depending on selected network name
+export const getStoragePrefix = (networkName = getNetworkName()): string => `_immortal|v2_${networkName}__`
 
 export const loadFromStorage = async <T = unknown>(
```

Comment:
Need this still be async?

Reply:
The code that uses it still expects a promise.

Reply:
Do you not think it's worthwhile changing it? Or create a new issue to do that?

Reply:
Yes, we should eventually un-async all that code. 👍
Created #3073